tsweb/tsweb.go

```diff
@@ -296,14 +302,23 @@ func (h retHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			}
 			lw.WriteHeader(msg.Code)
 			fmt.Fprintln(lw, hErr.Msg)
+			if hErr.RequestID != "" {
+				fmt.Fprintln(lw, hErr.RequestID)
+			}
 		}
 	case err != nil:
+		errorMessage := "internal server error"
+
+		if h.opts.GenerateRequestID != nil {
+			msg.RequestID = h.opts.GenerateRequestID(r)
+			errorMessage = errorMessage + "\n" + msg.RequestID
```

Comment:
Yeah, the docs on the type might mention it's appended after "internal server error\n" so should have a prefix that sounds like a useful identifier for humans reporting bugs. (Having just a number or just hex would look weird.)

Reply:
Included in the docs.